src/BuildScriptGeneratorCli/Commands/ExecCommand.cs

```diff
@@ -15,23 +17,65 @@
 using Microsoft.Oryx.BuildScriptGenerator;
 using Microsoft.Oryx.BuildScriptGenerator.Common;
 using Microsoft.Oryx.BuildScriptGenerator.Common.Extensions;
+using Microsoft.Oryx.BuildScriptGeneratorCli.Commands;
 using Microsoft.Oryx.BuildScriptGeneratorCli.Options;
 
 namespace Microsoft.Oryx.BuildScriptGeneratorCli
 {
-    [Command(Name, Description = "Execute an arbitrary command in the default shell, in an environment " +
-        "with the best-matching platform tool versions.")]
     internal class ExecCommand : CommandBase
     {
         public const string Name = "exec";
```

Comment:
Nit: Per https://github.com/microsoft/Oryx/pull/1816/files#r1129788587 I would lean towards keeping this and using it in the Command creation.